packages/ui-components/src/chat/chat.tsx

```diff
@@ -104,6 +104,8 @@ export interface ChatProps extends InteractiveComponentProps<Message[]> {
     isPopup?: boolean;
     /** A component showing the loading state of the chat, it appears above the input area, when not loading the caller can set it to null */
     loadingComponent?: React.ReactNode;
+    /** Whether the user can edit/delete previous messages */
+    nonEditable?: boolean;
```

Comment:
Would it be clearer to name `isHistoryEditable`? Prefixing with `is` follows the boolean naming convention and adding history emphasises that we can e.g. write new messages - it's just the history (i.e. past messages) which is not editable.